resources/schemas/org.gnome.shell.extensions.rounded-window-corners-reborn.gschema.xml

```diff
@@ -35,8 +35,13 @@
       <default>0</default>
     </key>
 
-    <key name="border-color" type="(dddd)">
-      <summary>Border color for rounded corners window</summary>
+    <key name="focused-border-color" type="(dddd)">
+      <summary>Border color for focused rounded corners window</summary>
```

Comment:
I think you should re-use the existing `border-color` pref and use it for both border colors by default, and then have a separate `focused-border-color` pref that acts as an override if it's set to a non-null value